bucket/vt-uploader.json

```diff
@@ -0,0 +1,49 @@
+{
+    "version": "0.1.9",
+    "homepage": "https://github.com/SamuelTulach/VirusTotalUploader",
+    "description": "A simple application written in C# that allows for file uploading to the VirusTotal website. (Requires a Virustotal API Key)",
+    "license": "GPL-3.0",
+    "notes": [
+        "You can add VirusTotalUploader as a context menu option by running,",
+        "reg import \"$dir\\install-context.reg\"",
+        "",
+        "For a guide on how to get your Virustotal API Key, visit,",
+        "https://github.com/SamuelTulach/VirusTotalUploader#how-to-get-api-key"
+    ],

```

Comment:
```suggestion
    "notes": [
        "For a guide on how to get your Virustotal API Key, visit,",
        "https://github.com/SamuelTulach/VirusTotalUploader#how-to-get-api-key"
    ],

```

Reply:
Why would I get rid of the notes that tell the user how to add `vt-uploader` as a context menu option.

Reply:
because it does not require elevated privileges now